api/src/main/java/org/eclipse/microprofile/concurrent/ThreadContextConfig.java

```diff
@@ -19,27 +19,42 @@
 package org.eclipse.microprofile.concurrent;
 
 import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 import java.lang.annotation.Retention;
 import java.lang.annotation.Target;
 
 /**
- * <p>Configures an injected <code>ThreadContext</code> instance.</p>
+ * <p>Provides configuration for an injected {@link ThreadContext} instance,
+ * which is {@link javax.enterprise.context.Dependent dependent} scoped.
+ * When an application has multiple injection points for {@link ThreadContext}
+ * with matching configuration, the container must inject the same instance.
+ * For the purposes of matching, array attributes of this annotation are
+ * considered as unordered sets where duplicate elements are ignored.</p>
  *
  * <p>Example usage:</p>
  * <pre><code> &commat;Inject &commat;ThreadContextConfig(ThreadContext.CDI)
  * ThreadContext threadContext;
  * ...
  * </code></pre>
  *
+ * <p>A {@link ThreadContext} service's life cycle is tied to that of the application.
```

Comment:
Similar comment as above. Also here is a mention of a `service` while there is no such mention with ManagerExecutor - we should either speak about services in both cases or in neither. Personally I wouldn't mention it as a service.

Reply:
Agree.  I think 'service' came from thinking of EE Concurrency's ContextService, which parallel's MicroProfile Concurrency's ThreadContext, but I'll remove the term because this is the MicroProfile spec, not the EE one.